experimenter/tests/integration/nimbus/pages/base.py

```diff
@@ -25,3 +25,10 @@ def _wait_for_it(selenium):
                 return True
 
         self.wait.until(_wait_for_it, message=message)
+
+    def find_element(self, strategy, locator):
+        self.wait.until(
+            EC.presence_of_element_located((strategy, locator)),
+            message=f"Element {locator} not found within the timeout period.",
```

Comment:
I noticed you are using `presence_of_element_located` which may be taking more time to run because it waits for the element to be present in the DOM, you can try using `visibility_of_element_located`, which waits for the element to be visible on the page rather than just present in the DOM, may be this will help cutting the time